src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -12,9 +12,31 @@ import {
   evidenceTypesDescription,
   EvidenceTypeHelp,
   disabilityNameTitle,
+  facilityDescription,
+  treatmentView,
   vaMedicalRecordsIntro
 } from '../helpers';
 
+const {
+  treatments
+} = fullSchema526EZ.properties;
+
+const {
+  date
+} = fullSchema526EZ.definitions;
+
+// We may add these back in after the typeahead, but for now...
+const treatmentsSchema = _.cloneDeep(treatments);
+treatmentsSchema.items.properties.treatment.properties = _.omit(
```

Comment:
Looks like we'll need to use these for private records, but we can handle this in that PR.

Reply:
Would it make sense to move this logic to the backend schema as done with 22-5490 and 21-526EZ?

Reply:
Nah, we're going to have to change this anyhow. I'm not sure how quite yet, though...